### PR TITLE
Create TSV file for non Recalibrated BAMs

### PR DESCRIPTION
**Pull request by MaxUlysse**

### CODE REVIEW

File: MultiFQtoVC.nf [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
+ Sebastian DiLorenzo <sebastian.dilorenzo@bils.se> [@Sebastian-D]
  Jesper Eisfeldt <jesper.eisfeldt@scilifelab.se> [@J35P312]
  Maxime Garcia <maxime.garcia@scilifelab.se> [@MaxUlysse]
  Szilveszter Juhos <szilveszter.juhos@scilifelab.se> [@szilvajuhos]
- Max Käller <max.kaller@scilifelab.se>
+ Max Käller <max.kaller@scilifelab.se> [@gulfshores]
  Malin Larsson <malin.larsson@scilifelab.se> [@malinlarsson]
  Björn Nystedt <bjorn.nystedt@scilifelab.se> [@bjornnystedt]
  Pall Olason <pall.olason@scilifelab.se> [@pallolason]
+ Pelin Sahlén <pelin.akan@scilifelab.se> [@pelinakan]
```
Comment (Sebastian-D) [inline MultiFQtoVC.nf]: He changed his name?

File: MultiFQtoVC.nf [hunk 2]
```diff
@@ -7,14 +7,15 @@
  New Cancer Analysis Workflow. Started March 2016.
 
  @Authors
- Pelin Akan <pelin.akan@scilifelab.se> [@pelinakan]
+ Sebastian DiLorenzo <sebastian.dilorenzo@bils.se> [@Sebastian-D]
```
Comment (Sebastian-D) [inline MultiFQtoVC.nf]: 👍

File: README.md [hunk 3]
```diff
@@ -6,14 +6,15 @@ Nextflow Cancer Analysis Workflow Prototype developed at @SciLifeLab
 0.0.3
 
 ## Authors
-- Pelin Akan (@pelinakan)
+- Sebastian DiLorenzo (@Sebastian-D)
```
Comment (Sebastian-D) [inline README.md]: 👍

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+## Tools and dependencies
+https://sourceforge.net/projects/bio-bwa/files/
+- bwa 0.7.8
+https://software.broadinstitute.org/gatk/download/index
+- GATK 3.3-0
+- GATK 3.6 [MuTect2]
+https://sites.google.com/site/strelkasomaticvariantcaller/home/download
+- Strelka 1.0.15
+- gcc 4.9.2
+- java sun_jdk 1.8.0_92
```
Comment (szilvajuhos) [inline INSTALL.md]: We should stick to java sun_jdk 1.8.0_40 only for now, as irma has only this version of 1.8

Comment (MaxUlysse) [inline INSTALL.md]: OK for me,
I should not have commited this file in my PR, but it's a start

**Pull request merged**